packages/gatsby-source-vtex/src/staticPaths.ts

```diff
@@ -100,8 +100,10 @@ const staticPaths = async ({
       fetchVTEX<Array<{ linkText?: string }>>(
         api.catalog.category.search({
           sort: 'OrderByTopSaleDESC',
-          from: pageIndex * itemsPerPage,
-          to: pageIndex * itemsPerPage + itemsPerPage,
+          from: pageIndex ? pageIndex * itemsPerPage + 1 : pageIndex,
```

Comment:
isn't it easier to make something like:
```suggestion
          from: pageIndex * itemsPerPage,
          to: pageIndex * itemsPerPage + itemsPerPage - 1,
```